samples/kfp-user-guide/README.md

```diff
@@ -79,13 +79,37 @@ Now, we should able to see the pipeline is being uploaded to the **Pipelines** p
 
 ### 2. Upload Pipelines Using the `kfp_tekton.TektonClient` in Python
 
-To upload the pipelines using Python, run the below code block inside the Python session. The below code block shows how to upload different versions of the pipeline using the Python client.
+To begin, we first need to declare our TektonClient:
+- For single user:
 ```python
-import kfp_tekton
-import os
-
+from kfp_tekton import TektonClient
 # **Important**: Replace None to the KFP endpoint if the python session is not running on the Kubeflow cluster.
 host = None
+KUBEFLOW_PROFILE_NAME = None
+client = TektonClient(host=host)
+```
+
+- For multi tenant:
+1. `KUBEFLOW_PUBLIC_ENDPOINT_URL` - Kubeflow public endpoint URL. 
```

Comment:
Shouldn't endpoint URL be a requirement for both single/multi-user?

Reply:
This is a limitation from kfp.
https://www.kubeflow.org/docs/pipelines/multi-user/#when-using-the-sdk
`First, you need to connnect to the Kubeflow Pipelines public endpoint using the SDK. `